commit 0b5512241a899663d88039694caac3144657d511 (refs/changes/17/3517/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-02-28T20:26:54+00:00 (1 year, 7 months ago)
    
    MANATEE-346 Ship both Postgres 9.2 and 9.6 in sdc-manatee images (need MANATEE-350)
    Reviewed by: Kody A Kantor <kody.kantor@gmail.com>
    Approved by: Kody A Kantor <kody.kantor@gmail.com>

diff --git a/pg_dump/pg_backup_common.sh b/pg_dump/pg_backup_common.sh
index c329a00..ac95e14 100644
--- a/pg_dump/pg_backup_common.sh
+++ b/pg_dump/pg_backup_common.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 # common functions used by postgres backup scripts
@@ -16,7 +16,7 @@ export PS4='[\D{%FT%TZ}] ${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}
 set -o xtrace
 set -o pipefail
 
-PATH=/opt/smartdc/manatee/build/node/bin:/opt/local/bin:/usr/sbin/:/usr/bin:/usr/sbin:/usr/bin:/opt/smartdc/registrar/build/node/bin:/opt/smartdc/manatee/node_modules/.bin:/opt/smartdc/registrar/node_modules/.bin:/opt/smartdc/manatee/lib/tools:/opt/smartdc/manatee/pg_dump/
+PATH=$PATH:/opt/smartdc/manatee/node_modules/.bin:/opt/smartdc/manatee/pg_dump/
 
 FATAL=
 CFG=/opt/smartdc/manatee/etc/backup.json
